app/backfill-synonym-slugs.js

```diff
@@ -0,0 +1,67 @@
+import { DynamoDBClient, paginateScan } from '@aws-sdk/client-dynamodb'
+import { GetParameterCommand, SSMClient } from '@aws-sdk/client-ssm'
+import { BatchWriteCommand } from '@aws-sdk/lib-dynamodb'
+import { unmarshall } from '@aws-sdk/util-dynamodb'
+import { slug } from 'github-slugger'
+import chunk from 'lodash/chunk'
+
+async function getTableNameFromSSM(dynamoTableName) {
+  const ssmClient = new SSMClient({ region: 'us-east-1' })
+
+  try {
+    const command = new GetParameterCommand({ Name: dynamoTableName })
+    const response = await ssmClient.send(command)
+
+    if (!response.Parameter?.Value) {
+      throw new Error('dynamoTableName not found in SSM')
+    }
+
+    return response.Parameter.Value
+  } catch (error) {
+    console.error('Error fetching table name from SSM:', error)
+    throw error
+  }
+}
+
+export async function backfillSlugsOnSynonyms() {
+  const startTime = new Date()
+  console.log('Starting SYNONYM SLUG backfill... ', startTime)
```

Comment:
Consider [`console.time`](https://developer.mozilla.org/en-US/docs/Web/API/console/time_static) and [`console.timeEnd`](https://developer.mozilla.org/en-US/docs/Web/API/console/timeEnd_static).